pkg/controllers/metrics/pod/controller.go

```diff
@@ -71,18 +72,47 @@ var (
 			Objectives: metrics.SummaryObjectives(),
 		},
 	)
+	podBoundDurationSeconds = prometheus.NewHistogramVec(
```

Comment:
I have a question on metric cardinality that I'm trying to get an answer on before we close this out -- specifically: I'm trying to understand from the K8s side if we should be concerned about the total number of timeseries that this could produce or if we should just be concerned about the _active_ timeseries. 

If it's the latter, I don't think that we have a problem here. If it's the former, we'll have to do some more thinking about how we can add this metric -- or any metric that can have potentially undefined cardinality